ibc_data/contrasts/PainMovie.csv

```diff
@@ -0,0 +1,3 @@
+condition,movie_mental-pain

```

Comment:
Making sure this is correct and that I can add that `fixation` line to model the `all_events` contrasts.

Reply:
see https://github.com/hbp-brain-charting/public_analysis_code/blob/master/ibc_public/utils_contrasts.py#L1764
there are two other conditions;

Reply:
I added the two other conditions, hope it's better ?